cmd/soci-snapshotter-grpc/main.go

```diff
@@ -125,8 +125,19 @@ func main() {
 		log.G(ctx).WithError(err).Fatal(err)
 	}
 
-	if err := service.Supported(*rootDir); err != nil {
-		log.G(ctx).WithError(err).Fatalf("snapshotter is not supported")
+	if !cfg.SkipCheckSnapshotterSupported {
+		if err := service.Supported(*rootDir); err != nil {
+			log.G(ctx).WithError(err).Fatalf("snapshotter is not supported")
+		}
+		log.G(ctx).WithFields(logrus.Fields{
+			"version":  version.Version,
+			"revision": version.Revision,
+		}).Debug("snapshotter is supported")
+	} else {
+		log.G(ctx).WithFields(logrus.Fields{
+			"version":  version.Version,
+			"revision": version.Revision,
+		}).Warn("skipped snapshotter is supported check ")
```

Comment:
`.Info` instead of `.Warn` IMO since this only happens intentionally. Also extraneous space

Reply:
although intentional this is a warning as its something that has risks.
An analogy can be for a deprecated API we provide an warning.

Reply:
Ok, that's fair enough. No issue then.